tests/functions/test_origin_response.py

```diff
@@ -0,0 +1,118 @@
+from cdn_lambda.functions.origin_response.origin_response import lambda_handler
+import pytest
+
+
+def test_origin_response_rpm():
+    event = {
+        "Records": [
+            {
+                "cf": {
+                    "request": {
+                        "headers": {
+                            "original-uri": [
+                                {
+                                    "key": "original-uri",
+                                    "value": "www.example.com/some/repo/some-rpm.rpm",
+                                }
+                            ]
+                        }
+                    },
+                    "response": {"headers": {}},
+                }
+            }
+        ]
+    }
+
+    response = lambda_handler(event, context=None)
+
+    # an rpm should not contain a CacheControl header
+    with pytest.raises(KeyError):
+        print(response["headers"]["cache-control"])
+
+
+def test_origin_response_listing():
+
+    event = {
+        "Records": [
+            {
+                "cf": {
+                    "request": {
+                        "headers": {
+                            "original-uri": [
+                                {
+                                    "key": "original-uri",
+                                    "value": "www.example.com/some/repo/listing",
+                                }
+                            ]
+                        }
+                    },
+                    "response": {"headers": {}},
+                }
+            }
+        ]
+    }
+
+    response = lambda_handler(event, context=None)
+
+    # listing files should contain cache headers
+    assert response["headers"]["cache-control"] == [
+        {"key": "Cache-Control", "value": "max-age=600"}
+    ]
+
+
+def test_origin_response_repomd():
```

Comment:
You may want to look into pytest.mark.parametrize, it allows you to run the same test code multiple times with different inputs and expected values.  Right now this seems to have the same test duplicated a few times and only with different URI values.  A parametrized test fits well here.